docs/modules/ROOT/nav.adoc

```diff
@@ -1,53 +1,81 @@
-.Get Started
+.Get started
+// Get started, release notes & support info
 * xref:index.adoc[Overview]
-* xref:ask-ai.adoc[]
-* xref:get-started.adoc[Deploy a Cluster]
+* xref:ask-ai.adoc[Ask AI]
+* xref:release-notes.adoc[Release notes]
+* xref:get-support.adoc[Get support]
+
+.Deploy
+// Deploy options
+* xref:get-started.adoc[Deploy a cluster]
+* xref:lite-members.adoc[Deploy Lite Members]
 * xref:deploy-management-center.adoc[Deploy Management Center]
-* xref:scaling-upgrading.adoc[Scaling & Upgrading]
-* xref:air-gapped-env.adoc[Running in air-gapped environments]
-* xref:connect-outside-kubernetes.adoc[]
-* xref:scheduling-configuration.adoc[Scheduling Hazelcast Pods]
-* xref:serialization-configuration.adoc[Serialization]
-* xref:scaling-upgrading.adoc[Scaling & Upgrading]
-* xref:lite-members.adoc[Deploying Lite Members]
-* xref:cp-subsystem.adoc[Enabling CP Subsystem]
-* xref:tiered-storage.adoc[Tiered Storage]
-* Configuring Data Structures
-** xref:map-configuration.adoc[Map]
-** xref:multimap-configuration.adoc[MultiMap]
-** xref:topic-configuration.adoc[Topic]
-** xref:queue-configuration.adoc[Queue]
-** xref:cache-configuration.adoc[Cache]
-** xref:vector-collection-configuration.adoc[Vector Collection]
-** xref:replicatedmap-configuration.adoc[ReplicatedMap]
-* xref:backup-restore.adoc[Persistence, Backup, and Restore]
-* xref:wan-replication.adoc[Configuring WAN Replication]
-* xref:wan-sync.adoc[Syncronizing WAN replicated data]
-* xref:user-code-deployment.adoc[User Code Deployment]
-* xref:high-availability-mode.adoc[High Availability Mode]
-* xref:native-memory.adoc[Native Memory]
-* xref:jvm-parameters.adoc[Configuring JVM Parameters]
-* xref:env-vars.adoc[Configuring Environment Variables]
-* xref:resource-configuration.adoc[Configuring Resource Limits]
-* xref:hazelcast-parameters.adoc[Configuring System Properties]
-* xref:advanced-networking.adoc[Advanced Networking]
-* xref:tls.adoc[Configuring TLS]
-* xref:user-code-namespaces.adoc[User Code Namespaces]
-* xref:authorization.adoc[Authorization Methods to Access Cloud Storage]
-* Data Pipelines
-** xref:jet-engine-configuration.adoc[Configuring the Jet Engine]
-** xref:jet-job-configuration.adoc[Running Data Pipelines]
-** xref:jet-job-snapshot.adoc[Saving the State of Data Pipelines]
-* xref:custom-config.adoc[Custom Configuration of Hazelcast cluster]
+* xref:flow.adoc[Deploy Flow]
+* xref:air-gapped-env.adoc[Run in air-gapped environments]
+* xref:scaling-upgrading.adoc[Scale and upgrade]
+
+.Configure
+// Configuration options
+* xref:connect-outside-kubernetes.adoc[Connect to Hazelcast from outside Kubernetes]
+* xref:scheduling-configuration.adoc[Assign Pods to nodes]
+
+* Memory & storage
+** xref:native-memory.adoc[High-density memory store]
+** xref:tiered-storage.adoc[Tiered storage]
+
+* Fault tolerance
+** xref:backup-restore.adoc[Backup & restore]
```

Comment:
```suggestion
* xref:backup-restore.adoc[Persistence, backup & restore]
```
I would go with this for now and then when we expand and change the structure we could bring the Fault tolerance back in in line with Platform. As we don't want top level nav items with only 1 child.